microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -53,6 +50,14 @@
 # declare package dependencies for package manager DPDK installation
 DPDK_PACKAGE_MANAGER_PACKAGES = DependencyInstaller(
     requirements=[
+        # install linux-modules-extra-azure if it's available for mana_ib
+        # bionic will never have mana/mana_ib, so skip in that case
+        OsPackageDependencies(
+            matcher=lambda x: isinstance(x, Debian)
```

Comment:
Because it checks `bionic`, does it apply to Ubuntu only? Check distro version is easier read than code name.

Reply:
Hmm.. I added it only for Ubuntu and have tested it with Ubuntu and older debians already. I don't think Debian has the linux-modules-extra-azure package but it's probably safe to enable it for Ubuntu *.*.*

Reply:
Ok! Tests show kernel version works best. I'm keeping it 🙂

Reply:
You don't need to use "kernel" version. The codename is the same as "distro" version. But the combination of "Debian" and "Ubuntu codename/distro version" doesn't make sense.

Reply:
Right, I had used codename just because I was testing bionic, buster, focal and jammy. But the issue is really the kernel version, so it's bad to rely on codename or even distro version. Because theoretically someone could backport MANA to an older kernel version.

Oh, I should update the comment so it's not confusing.